static/js/components/UserInfoCard.js

```diff
@@ -21,6 +22,13 @@ export default class UserInfoCard extends React.Component {
 
   render() {
     const { profile, toggleShowPersonalDialog } = this.props;
+    let emailInfo =  <div>
+      <Icon name="email" className="email-icon" />
+      <span className="profile-email">{profile.email}</span>
+    </div>;
+    if(SETTINGS.user !== undefined && SETTINGS.user.username === profile.username){
+      emailInfo = "";
+    }
```

Comment:
Amir just implemented a function called `isProfileOfLoggedInUser` in `util/util.js` which you could use here - it might be a little nicer if you pull the email info out into a helper method, and then below in the JSX you could do:

```js
{ isProfileOfLoggedInUser(profile) ? null : this.emailInfo() }
```

thoughts?